tests/fields/fields.go

```diff
@@ -17,13 +17,13 @@
 package main
 
 import (
+	"code.google.com/p/goauth2/oauth"
```

Comment:
Can you try to manually move it below and re-run `gofmt` (or `goimports`) to see if it is happy with it?  If so, I would leave it below the line break.